### ISSUE TITLE
Why don't use attention_mask in prepare_pairwise_sentence_embeddings.py#L153

### ISSUE THREAD

**Issue by chansonzhang** [kelvinlo-uni/Transformer-squared#1]
Hi, I noticed you don't use attention_mask here. Is there any specical reason for that? Thanks.
```python
output = Model(
  encoding['input_ids'], 
  token_type_ids=encoding['token_type_ids'], 
  output_hidden_states=True, 
  return_dict=True)
```
[prepare_pairwise_sentence_embeddings.py#L153](https://github.com/kelvinlo-uni/Transformer-squared/blob/638b2866a81a1e8ac3d1e9f8a88c2f8b1a290808/prepare_pairwise_sentence_embeddings.py#L153)

**Comment by kelvinlo-uni** [comment]
Hi chansonzhang,
Thanks for the question. We are encoding 1 pair of sentence at a time, so the model will automatically apply attention mask. (https://github.com/huggingface/transformers/blob/6c1d0b069de22d7ed8aa83f733c25045eea0585d/src/transformers/models/bert/modeling_tf_bert.py#L881-L882)
However, it is essential to also use the attention mask when we do batch inference with multiple sentence pairs of different lengths

**Comment by chansonzhang** [comment]
@kelvinlo-uni  thanks for your quick reply, it really helps.
